nlp_profiler/granular_features.py

```diff
@@ -41,6 +42,7 @@ def apply_granular_features(heading: str,
         (PUNCTUATIONS_COUNT_COL, text_column, count_punctuations),
         (STOP_WORDS_COUNT_COL, text_column, count_stop_words),
         (DATES_COUNT_COL, text_column, count_dates),
+        (NOUN_PHASE_COUNT_COL, text_column, count_phase)
```

Comment:
Could you please make the name consistent: `count_phase` to `count_noun_phase`, as we may have other kinds of phrase counts as well eventually.

Reply:
Sure I'll update it to count_noun_phase